.gitignore

```diff
@@ -74,6 +74,7 @@ typings/
 
 # parcel-bundler cache (https://parceljs.org/)
 .cache
+.parcel-cache
```

Comment:
- Can you remove `.parcel-cache` from this `.gitignore`
- Then delete the `.parcel-cache` folder. 
- Then commit that change
- Then add `.parcel-cache` back to the `.gitignore`
- Then add that change

Reply:
will do 👍